stubs/AMQPDecimal.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * stub class representing AMQPDecimal from pecl-amqp
+ */
+final class AMQPDecimal
+{
+    const E_MIN = 0;
```

Comment:
Yeah, will do. I think renaming `getE` to `getExponent` and `getN` to `getSignificand` will also a good thing.